### ISSUE TITLE
clone svn repository example

### ISSUE THREAD

**Issue by onny** [cespedes/svn#1]
hey,
looking for a native svn client in go to fetch a svn repository without relaying on subversion svn command to be installed on the system.

is cloning a svn repository already possible?

thank you and best regards
jonas

**Comment by cespedes** [comment]
No, not yet, sorry.  This is in a very early stage, and unfortunately it is not one of my top priorities right now, so don't expect to have it working in at least a few months.

I am leaving this issue open so that I will remember to make it work.

Thanks for your support!

Regards,
Juan
